ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -29,12 +29,13 @@
           </v-btn>
         </v-col>
         <v-col style="padding-top: 2px;">
-          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild }">Pending</p>
+          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild}">Pending</p>
           <!-- child drafts will sometimes show outside their base reg during the sort -->
           <div v-if="isChild || (isDraft(item) && item.baseRegistrationNumber)" :class="isChild ? 'pl-9' : ''">
-            <p class="ma-0">{{ item.registrationNumber }}</p>
+            <p v-if="!isPpr" style="font-size: 0.875rem !important;" :class="{ 'ma-0': true }">{{ item.documentId }}</p>
+            <p v-if="isPpr" class="ma-0">{{ item.registrationNumber }}</p>
```

Comment:
Maybe a good chance to use `v-if` and `v-else` 
v-if isPpr / v-else you doc id. 

Also, your styling in the `!isPpr` is a little off. Your bound class is always `true`? 
Also, is the `important` required? `p` default should be .875rem so if something is overriding, perhaps make a class to keep the template neat.

Reply:
Good point..